### ISSUE TITLE
ClientSidePageControls.cs FromHtml is not getting full HTML of the text control

### ISSUE THREAD

**Issue by Saulius-Laurinaitis** [SharePoint/PnP-Sites-Core#1421]
Hi,

there is an issue in ClientSidePageControls.cs function FromHtml line 399:
if ((div.FirstChild as IElement).TagName.Equals("P", StringComparison.InvariantCultureIgnoreCase))
{
      this.Text = (div.FirstChild as IElement).InnerHtml;
}
else
{
      this.Text = div.InnerHtml;
}

My text element on root level contains more that one element, e.g. first can be P, but then I have DIV, etc. So in this situation only first P is retrieved and remaining are just ignored. Example could be: add simple text then on new line add Pull Quote.

Fix would be just to delete if block and leave:
this.Text = div.InnerHtml;

#### Category
[ x] Bug

#### Environment
[x ] Office 365 / SharePoint Online

Thanks
Saulius

**Comment by mdissel** [comment]
Related to this issue is the ClientSideText.ToHtml() method, there a P tag is added by default
see https://github.com/SharePoint/PnP-Sites-Core/blob/master/Core/OfficeDevPnP.Core/Pages/ClientSidePageControls.cs#L368

**Comment by Saulius-Laurinaitis** [comment]
yes, but the issue I have is related to multiple div elements on the same root level and current code only takes the first one, so all other are just ignored. That's why my fix suggestion is to take inner html with all content or you can also make an array of HTMLElement where you could have all. Thanks.
